python/src/opendp/polars.py

```diff
@@ -24,6 +24,10 @@
 )
 from opendp.domains import series_domain, lazyframe_domain, option_domain, atom_domain
 from opendp.measurements import make_private_lazyframe
+from opendp._data import set_opendp_lib_path
+
+if lib_path is not None:
+    set_opendp_lib_path(str(lib_path))
```

Comment:
```suggestion
if lib_path is not None: # Should only be None when OPENDP_HEADLESS is set.
    set_opendp_lib_path(str(lib_path))
```
Is my understanding correct?